src/pipecat/services/aws/llm.py

```diff
@@ -967,7 +975,9 @@ async def _process_context(self, context: AWSBedrockLLMContext):
             }
 
             # Add system message
-            request_params["system"] = context.system
+            system = getattr(context, "system", None)
```

Comment:
This was a bug I discovered along the way. When we upgrade from `OpenAILLMContext` to `AWSBedrockLLMContext` we never properly initialize the `system` field. This felt like the lowest-risk way to address the problem.

This will be moot once we migrate to the universal `LLMContext`.